org-configurations/default-configuration.yml

```diff
@@ -0,0 +1,88 @@
+---
+name: "default name"
+slug: "default-slug"
+
+# configuration variables for the server app
+server:
+  host: "default-host"
+  proxy_urls:
+    password_change: "password_change_url"
+    password_reset: "password_reset_url"
+    password_reset_confirm: "password_reset_confirm_url"
+    registration: "registration_url"
+    user_auth_token: "user_auth_token_url"
+
+# configuration variables for the client
+client:
+  languages:
+    - "en"
+    - "it"
+  default_language: "en"
+  components:
+    header:
+      logo:
+        url: null # logo url
+        alternate_text: null
+      links:
+        - text:
+            en: "link1" #link text in english
+            it: null #link text in Italian
+          url: "url1"
+        - text:
+            en: "link2"
+            it: null
+          url: "link2url"
+
+    footer:
+      links:
+        - text:
+            en: "link1"
+            it: null
+          url: "url1"
+        - text:
+            en: "link2"
+            it: null
+          url: "link2url"
+
+    login_form:
+      input_field:
+        username:
+          type: "text"
+          placeholder:
+            en: "Enter phone number or username"
+            it: null
+          label:
+            en: "Phone number or username" # default username field label
+            it: null
+          regex: null #regex pattern for validation
+        password:
+          type: "password"
+          placeholder:
+            en: "Enter password"
+            it: null
+          label:
+            en: "Password" # default password field
+            it: null
+          regex: null
+      buttons:
+        login:
+          label:
+            en: null # label field text for login button
+            it: null
+          value:
+            en: 'Login' # text for login button
+            it: null
+        register:
+          label:
+            en: null # label field text for register button
+            it: null
+          value:
+            en: 'Sign up for free' # text for register button
+            it: null
+      links:
+        - forget_password :
+            en: "Forgot your password?"
+            it: null
+        - account:
+            en: "Manage your account"
+            it: null
```

Comment:
@nemesisdesign  We can separate configuration for each component to a different YAML file and write an extensive default config for each component which can be overridden by organisations. Should I go ahead with such configs and write a config for other components as well? Or we don't require this much customization in our app?
Vivek

Reply:
let's keep it simple for now, at max we can allow having one file for each organization if that does not complicate things too much